app/services/content_metadata_generator.rb

```diff
@@ -60,6 +60,7 @@ def create_file_node(id, cocina_file)
       file_node['publish'] = publish_attr(cocina_file)
       file_node['shelve'] = shelve_attr(cocina_file)
       file_node['preserve'] = preserve_attr(cocina_file)
+      cocina_file.key?('use') ? file_node['role'] = cocina_file.fetch('use') : nil
```

Comment:
```suggestion
      file_node['role'] = cocina_file.fetch('use') if cocina_file.key?('use')
```

Reply:
Suggested this, because the `nil` value from the ternary wasn't used.

Reply:
Done.